tests/util/skvbc.py

```diff
@@ -20,8 +20,9 @@
 
 WriteReply = namedtuple('WriteReply', ['success', 'last_block_id'])
 
+
 class SimpleKVBCProtocol:
-    KV_LEN = 21 ## SimpleKVBC requies fixed size keys and values right now
+    KV_LEN = 21  ## SimpleKVBC requies fixed size keys and values right now
```

Comment:
I see a bunch of pure formatting/spacing changes like this one. Can you please clean-up the PR by removing all of them, as they create unnecessary noise?

Thanks!